iothub/device/src/ConnectionStatusChangeReason.cs

```diff
@@ -8,13 +8,59 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public enum ConnectionStatusChangeReason
     {
+        /// <summary>
```

Comment:
Aren't you already making these changes in your other PR?

Reply:
Yes, I was; but as we discussed we want to put out non-breaking changes first -> which can then go into the LTS branch.
As a result, #1504 will now be broken into 2 PRs - #1506 - with no public API changes, and another one with public API changes (that I'll put up next).

Reply:
got it